lib/httplog/adapters/net_http.rb

```diff
@@ -35,5 +35,20 @@ def connect
 
       orig_connect
     end
+
+    private
+
+    def log_body(req, body=nil)
+      body.nil? || body.empty? ? log_body_stream(req.body_stream) : body
+    end
+
+    def log_body_stream(body_stream)
```

Comment:
nit: Ruby methods return `nil` by default so it could be simplified as:

```
if body_stream && body_stream.instance_variable_get('@stream').present?
  body_stream.instance_variable_get('@stream').seek(0)
  body_stream.to_s
end
```

Reply:
Thanks, done.